hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/CustomizedOpcodes.java

```diff
@@ -24,6 +24,7 @@
  * <p>Please see <a href="https://ethereum.org/en/developers/docs/evm/opcodes/">here</a>
  * for a review of the EVM opcodes.
  */
+@SuppressWarnings("ALL")
```

Comment:
Do we really want to suppress _all_ warnings here?

Reply:
added comments explaining the opcodes